Cargo.toml

```diff
@@ -169,6 +169,7 @@ members = [
     "rs/nervous_system/common/test_utils",
     "rs/nervous_system/humanize",
     "rs/nervous_system/integration_tests",
+    "rs/nervous_system/interaction",
```

Comment:
"interaction" sounds a bit generic. I wouldn't have guessed what it does from its name, especially the fact that it's supposed to be used not by canister code but mostly CLI tools. IMHO "ic-nervous-system-agent" is much clearer. WDYT?

Reply:
Good suggestion, done!